sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -150,6 +153,13 @@ private TenantConfiguration GetTestConfig(string name, Func<TenantConfiguration>
 
         public DateTimeOffset GetUtcNow() => Recording.UtcNow;
 
+        protected HttpPipelineTransport GetTransport() =>
+            new HttpClientTransport(
+                new HttpClient()
+                {
+                    Timeout = TimeSpan.FromSeconds(1000)
```

Comment:
Nit - move that into a constant.

Reply:
Is this a temporary change we are planning to revert?

Reply:
No, this is the the change that we said we would direct customers to make if running into timeouts.

Reply:
Nit - move that into a constant.